ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml

```
@@ -42,7 +42,10 @@ ref:
     documentation: "Specifies the logging level for terraform provider libraries."
   - name: SPOT_INSTANCES
     default: "false"
-    documentation:  "Use AWS Spot Instances for worker nodes.  Set to 'true' to opt into spot instances.  Explicitly set to 'false' to opt out.  Leave unset for the default, which may change."
+    documentation:  "Use AWS Spot Instances for *worker* nodes.  Set to 'true' to opt into spot instances.  Explicitly set to 'false' to opt out.  Leave unset for the default, which may change."
+  - name: SPOT_MASTERS
+    default: "false"
```

What about
```suggestion
    default: "true"
```
plus a `/hold` so we can rehearse this change for a while with capi/aws jobs and fix any issues? Then we can flip it back to `false` before merge and add the `SPOT_MASTERS=true` to the capi jobs definitions.